test/local/statsd_tests.js

```diff
@@ -0,0 +1,112 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var tap = require('tap')
+var test = tap.test
+
+var StatsDCollector = require('../../lib/metrics/statsd')
+var mockLog = {
+  error: function () {}
+}
+
+test(
+  'statsd init',
```

Comment:
nit: name this something like "statsd init failure cases" to distinguish it from the one below